client/my-sites/hosting/staging-site-card/index.js

```diff
@@ -212,6 +212,7 @@ export const StagingSiteCard = ( {
 
 	const { pushToStaging } = usePushToStagingMutation( siteId, stagingSite?.id, {
 		onSuccess: () => {
+			dispatch( recordTracksEvent( 'calypso_hosting_configuration_staging_site_push_success' ) );
 			setSyncError( null );
```

Comment:
[Suggestion] Change the naming to what this actually is: A "push started" event. (The same goes for pull event as well ).

So I suggest something like "calypso_staging_site_sync_push_started"

The "success" in the name is not correct, as this action can eventually fail.

If we want to track the success of this action, I propose adding a dispatch event [here](https://github.com/Automattic/wp-calypso/blob/b3229be4e9ddedc4f1d931548771728c3750cbd0/client/my-sites/hosting/staging-site-card/card-content/staging-sync-card.tsx#L417) as well.

Reply:
@kozer, thanks for the suggestion. I would prefer to leave it as is, to match the current `*_failure` tracks. I made clear what this event tracks using the `description` field of the registered event.